librz/core/cmd/cmd.c

```diff
@@ -5217,8 +5214,8 @@ struct ts_data_symbol_map map_ts_symbols[] = {
 /**
  * \brief Create an instance of RzCmd for the Rizin language
  */
-RZ_API RzCmd *rz_core_cmd_new(bool has_cons) {
-	RzCmd *res = rz_cmd_new(has_cons);
+RZ_API RzCmd *rz_core_cmd_new(RzCore *core, bool has_cons) {
```

Comment:
It would require updates of the plugins cc @wargio @thestr4ng3r

Reply:
not really, this is not called by the plugins.